indico/modules/events/registration/clone.py

```diff
@@ -0,0 +1,66 @@
+# This file is part of Indico.
+# Copyright (C) 2002 - 2015 European Organization for Nuclear Research (CERN).
+#
+# Indico is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 3 of the
+# License, or (at your option) any later version.
+#
+# Indico is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Indico; if not, see <http://www.gnu.org/licenses/>.
+
+from __future__ import unicode_literals
+
+from indico.core.db import db
+from indico.core.db.sqlalchemy.util.models import get_simple_column_attrs
+from indico.modules.events.registration.models.forms import RegistrationForm
+from indico.modules.events.registration.models.form_fields import RegistrationFormFieldData
+from indico.modules.events.registration.models.items import RegistrationFormItem, RegistrationFormSection
+from indico.util.i18n import _
+
+from MaKaC.conference import EventCloner
+
+
+class RegistrationFormCloner(EventCloner):
```

Comment:
I think we should also take care of the payment configuration, either here on in another cloner.